src/js/components/List/List.js

```diff
@@ -499,6 +500,7 @@ const List = React.forwardRef(
                     {...boxProps}
                     {...clickProps}
                     {...orderProps}
+                    {...defaultItemProps}
```

Comment:
Please check on the order in which this should be placed. I believe `defaultItemProps` will trump `itemProps` in this implementation, however, I believe we want the opposite.

Reply:
Yes I checked seemed like the ‘itemProps’ took over the ‘defaultItemProps’ I added an example in the story can you check there and make sure we’re on the same page?

Reply:
my mistake your correct..